### PR TITLE
Venezuela

### PR DESCRIPTION
**Pull request by Vesihiisi**
Mapping: https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/ve_(es)
Preview: https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/ve_(es)/preview

Task: https://phabricator.wikimedia.org/T174440

### CODE REVIEW

File: importer/Monument.py [hunk 1]
```diff
@@ -158,6 +158,19 @@ def add_statement(self, prop_name, value, quals=None, refs=None):
         statement = {"value": value, "quals": qualifiers, "refs": refs}
         base[prop].append(statement)
 
+    def has_statement(self, prop_name):
+        """
+        Check whether object has statement for given property.
```
Comment (lokal-profil) [inline importer/Monument.py]: missing an "a" before statement

File: importer/Monument.py [hunk 2]
```diff
@@ -158,6 +158,19 @@ def add_statement(self, prop_name, value, quals=None, refs=None):
...
+
+        :param prop_name: name of the property
+        """
+        base = self.wd_item["statements"]
+        prop = self.props["located_adm"]
+        if prop in base:
```
Comment (lokal-profil) [inline importer/Monument.py]: this can be simplified to `return (prop in base)`

File: importer/VeEs.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    def set_monuments_all_id(self):
+        self.monuments_all_id = self.id
+
+    def update_descriptions(self):
+        desc = "national historical monument of Venezuela"
+        if self.has_non_empty_attribute("ciudad"):
+            desc = desc + " in {}".format(utils.remove_markup(self.ciudad))
+        self.add_description("en", desc)
+
+    def exists_with_monument_article(self, language):
```
Comment (lokal-profil) [inline importer/VeEs.py]: See https://github.com/Vesihiisi/COH-tools/pull/76#discussion_r136988915

File: importer/VeEs.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+        if not self.has_statement("located_adm"):
+            state_dict = self.data_files["states"]
+            state_match = [x for x in state_dict if x[
+                "iso"] == self.estado_iso]
+            if len(state_match) == 1:
+                self.add_statement("located_adm", state_match[0]["item"])
+
+        if not self.has_statement("located_adm"):
+            self.add_to_report("municipio", self.municipio, "located_adm")
```
Comment (lokal-profil) [inline importer/VeEs.py]: should this not be reported even if we later manage to map a state?

File: importer/VeEs.py (hunk 3 above)
Comment (lokal-profil) [reply importer/VeEs.py]: Done

File: importer/Monument.py (hunk 1 above)
Comment (lokal-profil) [reply importer/Monument.py]: this whole method has been dropped

File: importer/Monument.py (hunk 2 above)
Comment (lokal-profil) [reply importer/Monument.py]: this whole method has been dropped

File: importer/VeEs.py (hunk 4 above)
Comment (lokal-profil) [reply importer/VeEs.py]: Done

**Comment by lokal-profil** [comment]
Preview updated to here. Matches running.

**Comment by lokal-profil** [comment]
Created [Matches](https://www.wikidata.org/w/index.php?title=Wikidata:WikiProject_WLM/Mapping_tables/ve_(es)/matches&oldid=588372973)

**Comment by lokal-profil** [comment]
Matches updated

**Comment by Vesihiisi** [comment]
All looks good. Cool matches......

**Pull request merged**